libkbfs/block_retrieval_queue.go

```diff
@@ -163,8 +169,84 @@ func (brq *blockRetrievalQueue) notifyWorker() {
 	}
 }
 
+func (brq *blockRetrievalQueue) CacheAndPrefetch(ptr BlockPointer, block Block,
+	kmd KeyMetadata, priority int, lifetime BlockCacheLifetime,
+	hasPrefetched bool) (err error) {
+	dbc := brq.config.DiskBlockCache()
+	if dbc != nil {
+		go dbc.UpdateMetadata(context.TODO(), ptr.ID)
```

Comment:
A linter might complain about ignoring the error here.  I'd prefer something that prints out unexpected errors, or at least assigns it to `_` so a linter would be satisfied.  But I won't fight too hard for it.